src/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ foreach(_source IN ITEMS ${PROJECT_SRC})
     message(STATUS "Including ${_source}")
 endforeach()
 
-add_library(project_lib ${PROJECT_SRC})
+add_library(project_lib ${PROJECT_SRC}
+        simulation/SimulationParams.cpp)
```

Comment:
This addition is useless i think. It is already captured in the GLOB_RECURSE above

Reply:
Ok